pkg/process/checks/process.go

```diff
@@ -430,11 +438,12 @@ func fmtProcesses(
 	connRates ProcessConnRates,
 	//nolint:revive // TODO(PROC) Fix revive linter
 	lookupIdProbe *LookupIdProbe,
+	zombiesIgnored bool,
```

Comment:
Nit: any reason not to rename this to `skipZombieProcesses`?

Reply:
For consistency purposes, decided to keep it as `ignoreZombieProcesses`